httpretty/__init__.py

```diff
@@ -55,6 +55,14 @@ def last_request():
     """returns the last request"""
     return httpretty.last_request
 
+def all_requests():
+    """Returns list of all requests made"""
+    return httpretty.latest_requests
```

Comment:
Yeah, it would me more consistent with underlying implementation, but what does it mean "latest_requests"? How many of them? :) all_requests is simple and understandable - just all of them...

But if you insist on another name, I don't care that much, I just use all_requests() everywhere in my code so I took it.